### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *stackblitz-docs* failed. **[Why did it fail? →](https://app.netlify.com/projects/stackblitz-docs/deploys/68b233866222f900073f4dba?deploy_diagnostics=true)**

### ISSUE THREAD

**Issue by jobayer1091** [returntocorp/sgrep-rules#3698]
### <span aria-hidden="true">❌</span> Deploy Preview for *stackblitz-docs* failed. **[Why did it fail? →](https://app.netlify.com/projects/stackblitz-docs/deploys/68b233866222f900073f4dba?deploy_diagnostics=true)**

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | df0628d4e720025193bb63b9af94c11d19caebc3 |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/projects/stackblitz-docs/deploys/68b233866222f900073f4dba |

_Originally posted by @netlify in https://github.com/stackblitz/docs/pull/241#issuecomment-3238649920_
